Gems/Atom/Tools/MaterialEditor/Code/Source/MaterialEditorApplication.h

```diff
@@ -92,28 +73,14 @@ namespace MaterialEditor
         void AssetSystemAvailable() override;
         //////////////////////////////////////////////////////////////////////////
 
-        //////////////////////////////////////////////////////////////////////////
-        // AZ::UserSettingsOwnerRequestBus::Handler overrides...
-        void SaveSettings() override;
-        //////////////////////////////////////////////////////////////////////////
-
         //////////////////////////////////////////////////////////////////////////
         // AZ::Debug::TraceMessageBus::Handler overrides...
```

Comment:
This is already enabled by the TraceLogger isn't it?  I think you can remove this bus and related functions.